CONTRIBUTING.md

```
@@ -0,0 +1,237 @@
+- **feat**: A new feature
+- **fix**: A bug fix
+- **docs**: Documentation changes
+- **refactor**: Code refactoring without feature/fix
+- **perf**: Performance improvements
+- **test**: Adding or updating tests
+- **chore**: Dependency updates or tooling changes
+
+## Pull Request Process
+
+1. **Push to your fork**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)